Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Provided resources: mem_mb=100000, mem_mib=95368, disk_mb=1000, disk_mib=954, nvidia_gpu=1
Select jobs to execute...

[Fri May 24 12:29:22 2024]
rule parameter_sampling:
    input: /ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/conf/CAMELYON_benchmark.yaml
    output: zoutput/benchmarks/CAMELYON_center0_vit_irm/hyperparameters.csv
    jobid: 1
    reason: Updated input files: /ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/conf/CAMELYON_benchmark.yaml
    resources: mem_mb=100000, mem_mib=95368, disk_mb=1000, disk_mib=954, tmpdir=/tmp, partition=gpu_p, qos=gpu_normal




!!!: not committed yet
/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/utils/get_git_tag.py:16: UserWarning: !!!: not committed yet
  warnings.warn("!!!: not committed yet")



b'44271c83'
[Fri May 24 12:29:30 2024]
Finished job 1.
1 of 2 steps (50%) done
Select jobs to execute...

[Fri May 24 12:29:30 2024]
rule run_experiment:
    input: zoutput/benchmarks/CAMELYON_center0_vit_irm/hyperparameters.csv
    output: zoutput/benchmarks/CAMELYON_center0_vit_irm/rule_results/7.csv
    jobid: 0
    reason: Input files updated by another job: zoutput/benchmarks/CAMELYON_center0_vit_irm/hyperparameters.csv
    wildcards: index=7
    resources: mem_mb=100000, mem_mib=95368, disk_mb=1000, disk_mib=954, tmpdir=/tmp, partition=gpu_p, qos=gpu_normal, nvidia_gpu=1

Failed to import backpack: cannot import name '_grad_input_padding' from 'torch.nn.grad' (/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/torch/nn/grad.py)
[Fri May 24 12:29:43 2024]
Error in rule run_experiment:
    jobid: 0
    input: zoutput/benchmarks/CAMELYON_center0_vit_irm/hyperparameters.csv
    output: zoutput/benchmarks/CAMELYON_center0_vit_irm/rule_results/7.csv

RuleException:
KeyError in file /ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/benchmark.smk, line 162:
7
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/benchmark.smk", line 162, in __rule_run_experiment
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/run_experiment.py", line 80, in run_experiment
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/run_experiment.py", line 28, in load_parameters
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1073, in __getitem__
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1312, in _getitem_axis
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1260, in _get_label
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/generic.py", line 4056, in xs
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3804, in get_loc
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
